src/models/v2/nftRewardTier.ts

```diff
@@ -28,10 +28,16 @@ export type ContractNftRewardTier = {
 // How the reward tiers are stored on IPFS
 // Tank - the following are guidelines on NFT keys and tier JSON, derivates from EIP-721, https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md
 export type IPFSNftRewardTier = {
-  attributes: {
-    contributionFloor: number
-    maxSupply: number | undefined
-  }
+  attributes: [
+    {
+      trait_type: string
+      value: number | undefined
+    },
+    {
+      trait_type: string
+      value: number | undefined
+    },
```

Comment:
**suggestion**: maybe we can have a type for this, and then `attributes` becomes something like

`attributes: Attribute[]`

Reply:
Yeah I wanted to do that, but the linter was screaming at me for `'trait_type'` not being camelcase

Reply:
want to ignore it just for that line then? seems fine